src/components/SSOLanding.js

```diff
@@ -42,12 +42,18 @@ class SSOLanding extends Component {
     const token = this.getToken();
 
     if (!token) {
-      return <div>No <tt>ssoToken</tt> cookie or query parameter</div>;
+      return (
+        <div>
+          {'No <tt>ssoToken</tt> cookie or query parameter'}
```

Comment:
[Here's a link to React docs on dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml). The concern with using HTML inside of strings (especially with template literals) is about XSS attacks

Reply:
What's the value of this change (and the others that are similar) to begin with? I don't see how the new code, with string interpolation that causes problems deep in React, is in any way superior to the code it replaced.

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-one-expression-per-line.md

The rule is designed to make JSX construction very deliberate.